new-dti-website/components/courses/data/trend_instructors.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Instead of storing the data of the Trends instructors, it would probably be easier in the future while editing to store the netids (then filter through the archive), in somewhere like `config.json`, which has data that changes semester to semester. Maybe consider that for `timeline_events.json` as well.

Reply:
> Instead of storing the data of the Trends instructors, it would probably be easier in the future while editing to store the netids (then filter through the archive), in somewhere like `config.json`, which has data that changes semester to semester. Maybe consider that for `timeline_events.json` as well.

Very minor concern is that we have had a few TA's/instructors in the past for trends be people who were not on DTI (ex. Fall 2021 and Spring 2022), so I'm not fully against decoupling these two and copy-pasting the information inside. That way we're not forced to find a "workaround" for when someone not on DTI joins the course staff.

I don't feel that strongly about it either way, but it seems leaving it as it is now would be less work, no need to put the additional logic in. What do you think? @JasonMun7 @cchrischen